### PR TITLE
West Midlands Class 3 - Peter Ahern - Git&GitHub - Week 1

### PR DESCRIPTION
**Pull request by PeterAhern**
I edited and changed the answers on the homework file, most of them were incorrect, so I ended up changing most of the answers.

# Your Details

- Your Name: Peter Ahern
- Your City: Birmingham (West Midlands)
- Your Slack Name: Peter Ahern

# Homework Details

- Module: Git & GitHub
- Week: 1

-----
[View rendered homework.md](https://github.com/PeterAhern/GitHomeworkFixErrors/blob/main/homework.md)

### CODE REVIEW

**Review by justsealan — CHANGES_REQUESTED** [review]
For question 6; if you write #h1 you are only selecting the elements with id = "h1". To select all the h1 elements, you can just write h1, no need to add anything.

File: homework.md [hunk 1]
```diff
@@ -50,31 +50,31 @@ A programming language used on the internet
...
 
-When you send a file over the internet
+Pull requests let you tell others about changes you've pushed to a branch in a repository on GitHub. Once a pull request is opened, you can discuss and review the potential changes with collaborators and add follow-up commits before your changes are merged into the base branch.
 
 ## Question 12
 
 What is the Git History log?
 
-It lets you see into the future
+It is a saved record of all the commits made to the repository to be able to track and understand changes made.
```
Comment (wiamnasr) [inline homework.md:77]: Also useful to revert back certain features, if needed

File: homework.md [hunk 2]
```diff
@@ -92,19 +92,19 @@ A place for us to store and share our code.
...
+Visual Studio Code
 
 ## Question 17
 
 What do you use "Commit Messages" for?
 
-To record what changes you made to the code
+To record what changes you made to the code, and explain the reason why you made them.
 
 ## Question 18
```
Comment (wiamnasr) [inline homework.md:110]: The answer to 'Where can I look to find help with Git?' should be here

**Review by wiamnasr — CHANGES_REQUESTED** [review]
Reviewed your request, hope this was helpful

**Pull request closed without merging**